dev/drivers/scripts/stats/analyses/jevs_analyses_ccpa_precip_stats.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#PBS -N jevs_analyses_ccpa_precip_stats
+#PBS -j oe
+#PBS -S /bin/bash
+#PBS -q "dev"
+#PBS -A VERF-DEV
+#PBS -l walltime=00:30:00
+#PBS -l place=shared,select=1:ncpus=1:mem=10GB
+#PBS -l debug=true
+#PBS -V
+ 
+set -x
+
+cd $PBS_O_WORKDIR
+
+export model=evs
+
+export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS
+
+source $HOMEevs/versions/run.ver
+
+evs_ver_2d=$(echo $evs_ver | cut -d'.' -f1-2)
+
+############################################################
+## Load modules
+#############################################################
+
+module reset
+module load prod_envir/${prod_envir_ver}
+
+
+############################################################
+## For dev testing
+#############################################################
+
+export envir=prod
+export DATAROOT=/lfs/h2/emc/stmp/${USER}/evs_test/$envir/tmp
```

Comment:
@PerryShafran-NOAA @ShelleyMelchior-NOAA Do we want these /evs_test directories to make their way into develop in the /dev/drivers?

Reply:
Yup.  I started with the grid2obs driver scripts when I made the precip scripts and just made modifications for precip.  It's definitely not new.